### PR TITLE
adding linked in url

### PR DESCRIPTION
**Pull request by Kim-Heilig**

### CODE REVIEW

**Comment by jacobboney** [comment]
Hey Kim, GitHub and version control as a whole does have a bit of a learning curve. However, you are using the platform properly. In practice though you'll only do PRs when you have successfully completed a feature, fix, or in this case an assignment. 

The concept of having branches is to prevent any system breaking changes from affecting the main production code. You will make all changes (commits) on your development branch. Once you have implemented all of your changes or additions to your codebase you will then make a pull request which will incorporate all of those changes from your development branch into your main production branch. 

The other added benefit of branches is that it allows multiple people to work on a codebase without their changes affecting the rest of the developers. 

So, in summary, make as many commits as you'd like to your branch and make a single PR when you are finished with the assignment. Hope that clears things up for you, let me know if you still have any confusion. If it makes you feel better I still get tripped up sometimes! Practice is your friend here.

As for your code, everything looks great and you're on track!
